src/services/expenses.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const { Expense } = require('../models/Expense');
+const { Op } = require('sequelize');
+
+const getWhereFromQuery = query => {
+  const where = {};
+
+  Object.entries(query).forEach(([key, value]) => {
+    switch (key) {
+      case 'from':
+        where.spentAt = {
+          [Op.gt]: value,
+        };
+        break;
+
+      case 'to':
+        where.spentAt = {
+          [Op.lt]: value,
+        };
+        break;
+
+      default:
+        where[key] = value;
+    }
+  });
+
+  return where;
+};
```

Comment:
If I'm not wrong, you are reassigning where.spentAt, so the user will not have the opportunity to select everything from 'date' to 'date'.

Reply:
you are indeed not wrong